.github/workflows/deploy-DEV.yml

```
@@ -82,4 +82,4 @@ jobs:
       ############################################
       - name: Run against all code base
         shell: bash
-        run: docker run -e RUN_LOCAL=true -e OUTPUT_DETAILS=detailed -v ${GITHUB_WORKSPACE}:/tmp/lint github/super-linter:${GITHUB_SHA}
+        run: docker run -e RUN_LOCAL=true -e OUTPUT_DETAILS=detailed -v ACTIONS_RUNNER_DEBUG=true -v ${GITHUB_WORKSPACE}:/tmp/lint github/super-linter:${GITHUB_SHA}
```

This should be `-e ACTIONS_RUNNER_DEBUG=true`, not -v. I think I introduced this error in a old PR :(

---

```suggestion
        run: docker run -e RUN_LOCAL=true -e OUTPUT_DETAILS=detailed -e ACTIONS_RUNNER_DEBUG=true ${GITHUB_WORKSPACE}:/tmp/lint github/super-linter:${GITHUB_SHA}
```

Like this @ferrarimarco

---

or do I keep the `-v` after the `ACTIONS_RUNNER_DEBUG`

---

keep it :) just change -v ACTIONS_RUNNER_DEBUG to -e ACTIONS_RUNNER_DEBUG

---

```suggestion
        run: docker run -e RUN_LOCAL=true -e OUTPUT_DETAILS=detailed -e ACTIONS_RUNNER_DEBUG=true -v ${GITHUB_WORKSPACE}:/tmp/lint github/super-linter:${GITHUB_SHA}
```